requirements.txt

```diff
@@ -12,6 +12,5 @@ python-neutronclient
 python-novaclient
 -e git+https://github.com/ging/python-keystoneclient.git#egg=python-keystoneclient
 
-requests!=2.9.0,>=2.8.1 # Apache-2.0
-
+requests
```

Comment:
Are you sure that we need these packages? docopt I doubt, requests I am not sure...